stellarphot/utils/magnitude_system_transforms.py

```diff
@@ -82,9 +84,42 @@ def polynomial(self) -> np.polynomial.Polynomial:
         return np.polynomial.Polynomial(self.polynomial_coefficients)
 
 
+class MagnitudeTransformMatrix(BaseModel):
+    """
```

Comment:
Do we want the same definition for both "MagnitudeTransformPolynomial" and this class?

Reply:
I wish -- in the long run we can do this because we can write the polynomial thing as a matrix if we decide we need to. For now it was easier to just make two classes to represent the two broad ways I've seen of transforming.